File: VideoGameCatalog.Models/Game.cs
namespace VideoGameCatalog.Models
{
    public class Game
    {
        public int Id { get; set; }

        public string Title { get; set; } = string.Empty;

        public string Genre { get; set; } = string.Empty;

        public decimal Price { get; set; }

        public DateTime ReleaseDate { get; set; }
    }
}

File: video-game-catalog-ui/src/app/models/game.ts
export interface Game {
  id: number;
  title: string;
  genre: string;
  price: number;
  releaseDate: string; // ISO date string from API
}

export interface CreateGameRequest {
  title: string;
  genre: string;
  price: number;
  releaseDate: string;
}

export interface UpdateGameRequest extends CreateGameRequest {
  id: number;
}